src/include/processor/operator/persistent/copy_to.h

```diff
@@ -29,6 +29,22 @@ struct CopyToLocalState {
     std::vector<std::shared_ptr<common::ValueVector>> inputVectors;
 };
 
+struct CopyToPrintInfo final : OPPrintInfo {
+    std::vector<std::string> exportNames;
+    std::string fileName;
+
+    CopyToPrintInfo(std::vector<std::string> exportNames, std::string fileName)
+        : exportNames{std::move(exportNames)}, fileName{std::move(fileName)} {}
+    CopyToPrintInfo(const CopyToPrintInfo& other)
```

Comment:
I recently realize that we probably don't need public copy constructor for these PrintInfo classes.  So make copy constructor private and change 
```
std::make_unique

to

std::unique_ptr<>(new CopyToPrintInfo())
```

Try apply this change to all printInfos.